client/src/main/java/gui/controllers/PlacementSceneController.java

```diff
@@ -61,9 +60,13 @@ public void initialize() {
 
     @FXML
     private void placeRandom() {
+        appInstance.getServerConnection().sendMessage("ASK_FOR_FLEET");
+
+        String message = appInstance.getServerConnection().getMessage();
+
         printingBoard.getChildren().removeIf(node -> node instanceof Shape);
-        FleetDAO fleetDAO = new RandomFleet();
-        fleet = fleetDAO.getGUIFleet();
+        RandomFleet generator = new RandomFleet();
```

Comment:
fleetGenerator maybe?
In next lines when you have only "generator" it is not that easy to say what it generates

Reply:
Corrected.